tools/msstats/msstats.xml

```diff
@@ -0,0 +1,782 @@
+            <option value="r_script" selected="false">MSstats Rscript</option>
+            <option value="processed_data" selected="true">MSstats ProcessedData</option>
+            <option value="runlevel_data" selected="false">MSstats RunlevelData</option>
+            <option value="qcplot" selected="true">MSstats QCPlot.pdf</option>
+            <option value="profile_plot" selected="false">MSstats ProfilePlot.pdf</option>
+            <option value="profile_wsum_plot" selected="false">MSstats ProfilePlot_wSummarization.pdf</option>
+            <option value="condition_plot" selected="false">MSstats ConditionPlot.pdf</option>
+            <option value="quant_sample_matrix" selected="false">Sample Quantification Matrix Table</option>
+            <option value="quant_sample_long" selected="false">Sample Quantification Long Table</option>
+            <option value="quant_group_matrix" selected="true">Group Quantification Matrix Table</option>
+            <option value="quant_group_long" selected="false">Group Quantification Long Table</option>
+        </param>
+
+        <conditional name="group">
+            <param name="group_comparison" type="select" label="Compare Groups">
+                <option value="no">No</option>
+                <option value="yes">Yes</option>
+            </param>
+            <when value="no"/>
+            <when value="yes">
+                <param name="comparison_matrix" type="data" format="tabular,csv" label="Comparison Matrix"/>
+                <param name="select_outputs" type="select" multiple="true" label="Select outputs">
+                    <help>Heatmap requires more than one comparison</help>
+                    <option value="fittedmodel" selected="true">MSstats ComparisonFittedModel.txt</option>
+                    <option value="comparison_result" selected="true">MSstats ComparisonResult.tsv</option>
+                    <option value="model_qc" selected="false">MSstats ModelQC.tsv</option>
+                    <option value="qqplot" selected="false">MSstats QQPlot.pdf</option>
+                    <option value="residualplot" selected="false">MSstats ResidualPlot.pdf</option>
+                    <option value="volcanoplot" selected="true">MSstats VolcanoPlot.pdf</option>
+                    <option value="heatmap" selected="false">MSstats Heatmap.pdf</option>
+                    <option value="comparisonplot" selected="true">MSstats ComparisonPlot.pdf</option>
+                </param>
+            </when>
+        </conditional>
+    </inputs>
+
+    <outputs>
+        <data name="log" format="txt" label="MSstats log">
+            <filter>'log' in selected_outputs</filter>
+        </data>
+        <data name="r_script" format="txt" label="MSstats Rscript">
+            <filter>'r_script' in selected_outputs</filter>
+        </data>
+        <data name="processed_data" format="tabular" label="MSstats ProcessedData" from_work_dir="ProcessedData.tsv">
```

Comment:
can we or should we add column names here and the other outputs, or do we not know the column names?